app/main/helpers/validation.py

```diff
@@ -238,4 +238,5 @@ class DOS2Validator(G8Validator):
     "g-cloud-8": G8Validator,
     "digital-outcomes-and-specialists": DOSValidator,
     "digital-outcomes-and-specialists-2": DOS2Validator,
+    "g-cloud-9": DOS2Validator,
```

Comment:
The declaration is currently exactly the DOS2 declaration and this will work.  It is effectively a placeholder to make G9 submissions work.  Once declaration is reviewed and changed it will force us to build a G9Validator.  

We already have
```
class DOS2Validator(G8Validator):
    pass
```

adding 
```
class G9Validator(DOS2Validator):
    pass
```

feels like overkill at this point.  And yes, I'm sure it will change in near future.